src/trie/trie_type.c

```diff
@@ -88,7 +88,7 @@ Vector *Trie_Search(Trie *tree, char *s, size_t len, size_t num, int maxDist, in
     }
     // in prefix mode we also factor in the total length of the suffix
     if (prefixMode) {
-      ent->score /= sqrt(1 + fabs(slen - len));
```

Comment:
Actually, it's not safe to cast size_t to an unsigned type, so now I'm just subtracting the lesser number from the greater number.